submissions/andrewklmn/html-css-popup/app.css

```diff
@@ -0,0 +1,253 @@
+* {
+  padding: 0px;
+  margin: 0px;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  /*
+  background-image: url(img/bg_image.jpg);
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center; 
+  */
+  width: 100vw;  
+  height: 100vh;
+}
+
+label.button {
+  color:black;
+  cursor: pointer;
+  border: white solid thin;
+  box-sizing: border-box;
+}
+
+label.button:hover {
```

Comment:
Better change to class. I write above about specific and u can read this article https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4